AppCenter/AppCenter/MSAppCenter.h

```diff
@@ -31,6 +38,14 @@
  */
 + (void)start:(NSString *)appSecret withServices:(NSArray<Class> *)services;
 
+/**
+ * Configure the SDK with an array of services to start.
+ *
+ * @discussion This may be called only once per application process lifetime.
```

Comment:
We prevent from starting AC from application multiple times already.

Reply:
I'd still mention this in the header file.

Reply:
Yes, still good to have.